Command/GenerateTokenCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Command;
+
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTTokenManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Security\Core\User\User;
+
+/**
+ * GenerateTokenCommand.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class GenerateTokenCommand extends Command
+{
+    protected static $defaultName = 'lexik:jwt:generate-token';
+
+    private $tokenManager;
+
+    public function __construct(JWTTokenManagerInterface $tokenManager)
+    {
+        parent::__construct();
+
+        $this->tokenManager = $tokenManager;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName(static::$defaultName)
+            ->setDescription('Generate a JWT token')
+            ->addArgument('username', InputArgument::REQUIRED)
+            ->addOption('user-class', 'c', InputOption::VALUE_REQUIRED, 'Class of the User to use.', User::class)
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $userClass = $input->getOption('user-class');
+        $token     = $this->tokenManager->create(new $userClass($input->getArgument('username'), null));
+
+        $output->writeln([
+            '',
+            '   <info>'.$token.'</info>',
```

Comment:
why did you put a space before the `<info>` tag?

Reply:
So it looks nice :)

Reply:
adding extra chars would prevent redirecting the output (e.g. to clipboard), I would be very raw here :)

Reply:
Meh, fair point.